Automattic/wp-calypso
# Jetpack Cloud: Add Jetpack Social promo screen

#### Changes proposed in this Pull Request

When Jetpack Social isn't installed then we will display a promo encouraging people to download it from the repo. Similarly if it's installed but Publicize isn't enabled, then we'll display the same screen, but link people to the admin page in order to enable it.

#### Testing instructions

- Check that with Jetpack installed, you don't see the promo at all, and either see the connections screen or a prompt to enable Publicize, which works.
- Check that with another standalone plugin installed you see the promo with a 'Get Started' link that takes you to the repo page
- Check that with Social installed, but Publicize disabled, that you see the promo with an 'Enable Social' link that takes you to the plugin's admin page.

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/96462/170353397-6da21ddf-b05b-42fc-b91a-d0fb9b5f7221.png">

The promo without Social or Jetpack installed.

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/96462/170353643-f5b8fc25-47a7-4362-9a52-110606f3360f.png">

The promo with Social installed, but Publicize disabled.
